newcomers-tutorial.md

```diff
@@ -319,7 +310,7 @@ and lead to further optimisation.
 -}
 ```
 
-Note that:
+Please pay attention to:
```

Comment:
This doesn't grammar: "Please pay attention to the Note itself is a long block of prose." Besides, I think "Note that" is perfectly appropriate - we're simply explaining the convention here, and "note that:" is a fairly common way of directing the reader's attention to some points we want to highlight. I don't see a reason to change that.

Reply:
Sure. I was trying to get rid of "note note" :-)

Reply:
BTW "this doesn't grammar" is great :)